### PR TITLE
Update contents.json

### PR DESCRIPTION
**Pull request by eugenebokhan**
Added "Awesome ML" open-source project for demonstrating different image-processing CoreML models on iOS devices.

1. [x] Project URL:
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Avoid iOS or open-source in description as it is assumed
5. [x] Use this commit title format if applicable: Add app-name by @github-username
6. [x] Use approved format for your entry

### CODE REVIEW

**Comment by dkhamsing** [comment]
Looks like your editor changed some regular quotes `"` to smart quotes `“` - could you change it?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: contents.json [hunk 1]
```diff
@@ -13076,6 +13076,16 @@
       "suggested_by": "@dkhamsing",
       "stars": 36
     },
+    {
+      "title": "Awesome ML",
+      "category-ids": ["developer"],
+      "description": "Discover, download, compile & launch different image processing CoreML models on iOS.",
```
Comment (vpeschenkov) [inline contents.json]: Could we use this: "Discover, download, compile & launch different image processing CoreML models" instead?

File: contents.json [hunk 2]
```diff
@@ -13076,6 +13076,16 @@
...
+      "source": "https://github.com/eugenebokhan/Awesome-ML",
+      "screenshots": ["https://github.com/eugenebokhan/Awesome-ML/raw/master/Media/App-Demo-Transparent-Background.gif"],
```
Comment (vpeschenkov) [inline contents.json]: Could you add the next line: `"license": "bsd-3-clause"`?

File: contents.json [hunk 3]
```diff
@@ -13076,6 +13076,16 @@
...
+      "date_added": "April 2 2018",
```
Comment (vpeschenkov) [inline contents.json]: Could you add the next line: `"tags": ["swift"]`?

File: contents.json [hunk 4]
```diff
@@ -13076,6 +13076,16 @@
...
+      "suggested_by": "@eugenebokhan"
```
Comment (vpeschenkov) [inline contents.json]: Looks like we missed comma at the end here `"suggested_by": "@eugenebokhan"`, could you add the one?

**Comment by eugenebokhan** [comment]
@vpeschenkov Hi! Thank you so much for your help!

**Comment by vpeschenkov** [comment]
@eugenebokhan thank you! 🙂

**Pull request merged**